engines/grim/grim.cpp

```diff
@@ -1117,6 +1117,10 @@ void GrimEngine::mainLoop() {
 			g_imuseState = -1;
 		}
 
+		#if defined(EMSCRIPTEN)
```

Comment:
there is already yield here. is playback too much slow?

Reply:
_I deleted my previous commit as it wasn't relevant anymore_

I once again looked into this, there's no way to avoid a forced yield here. The logic which is yielding here depends on the frame rate and the limit, which means that as soon as we don't hit the speed limit, the whole browser tab freezes (as the browser doesn't get control over it's main loop back).

Reply:
ok then